.circleci/scripts/binary_linux_build.sh

```diff
@@ -16,9 +16,7 @@ if [[ "${DESIRED_CUDA}" =~ cu1[1-2][0-9] ]]; then
 fi
 
 # Parse the parameters
-if [[ "$PACKAGE_TYPE" == 'conda' ]]; then
-  build_script='conda/build_pytorch.sh'
```

Comment:
Why not delete `conda/build_pytorch.sh` script as well?

Reply:
I believe its already deleted by : https://github.com/pytorch/builder/pull/2045